cloud/amazon/route53.py

```diff
@@ -511,7 +526,10 @@ def main():
     except boto.route53.exception.DNSServerError, e:
         txt = e.body.split("<Message>")[1]
         txt = txt.split("</Message>")[0]
-        module.fail_json(msg = txt)
+        if "but it already exists" in txt:
```

Comment:
My only concern with this is this error check. I don't like the idea of trying to match the text in an error message since that text could change at a moment's notice by Amazon and/or boto. If they did change the text of this error then it then it would cause a pretty significant change in the way that this module behaves.  What once returned successfully without error would suddenly trigger errors.  Is there no better way of performing this check?